website/README.md

```diff
@@ -228,34 +221,17 @@ Populate the `.env` file with the following keys and their respective values.
 
 **Note**: Remove `DATABASE_URI` variable  if you are using docker
 
-#### Docker
-
```

Comment:
Hi @FaithDaka , I notice quite a number of instructions being removed from the README, which is making me a bit anxious. 

What is the purpose of this PR again?

Thanks!

Reply:
They are not being removed. Just put in a different place so that users have a nice process while setting up the website. You'll notice these lines else where in the READMe (so it's just UX).
Please review using the _rich diff_ view to appreciate what I'm saying @Baalmart. Thanks!